app/controllers/application_controller.rb

```diff
@@ -14,6 +14,11 @@ def not_found
     raise ActiveRecord::RecordNotFound, "Not Found"
   end
 
+  def not_authorized
```

Comment:
This should probably be in the `ApiController` because it's supposed to return JSON for the API.

BTW this method weirdly returns 404 again, because `Pundit::NotAuthorizedError` has no registered exception handler, which results in a page not found for some reason. 

My suggestion is to use the same technique with `rescue_from`, remove the `render json` line and use an exception handler to render the json in this style `{"status":404,"error":"Not Found"}` so that the client can expect a single way to parse an error message

Reply:
Ah yeah, I suspected that too when I pushed it. Thanks for the feedback!